Changelog.md

```diff
@@ -2,6 +2,14 @@
 
 ### libsdformat 11.X.X (202X-XX-XX)
 
+### libsdformat 11.2.2 (2021-07-01)
+
+1. Fix segfault when checking for required elements in joint
+    * [Pull request #610](https://github.com/ignitionrobotics/sdformat/pull/610)
+
+1. Add ValidateGraphs methods to Model/World
+    * [Pull request #601](https://github.com/ignitionrobotics/sdformat/pull/601)
```

Comment:
add #575 which was merged forward in #615?

Reply:
I think @chapulina has a script for updating the changeling

Reply:
I didn't think we added PRs from merge forwards, but #575 isn't in the changelog because we haven't made a new release of libsdformat9, so adding it here makes sense.

Reply:
0aec1105a952e47bdf2a03562e29e698c576d29a

Reply:
I like adding them so we know which minor gets the features that are merged forward

otherwise it takes a lot of digging to figure out

Reply:
Thanks!

Reply:
> I think @chapulina has a script for updating the changeling

It's here: https://github.com/ignition-tooling/release-tools/issues/246#issuecomment-791833598

I usually change it manually for SDF because it's not prefixed by `ign`.

> I like adding them so we know which minor gets the features that are merged forward

+1